analysis/druid/src/ConvokeSpirits.tsx

```diff
@@ -329,15 +334,15 @@ class ConvokeSpirits extends Analyzer {
           <table className="table table-condensed">
               <thead>
                 <tr>
-                  <th>Cast #</th>
+                  <th>Cast #/form</th>
                   <th>Spells In Cast</th>
                 </tr>
               </thead>
               <tbody>
                 {
                   this.whatHappendIneachConvoke.map((spellIdToCasts, index) => (
                     <tr key={index}>
-                      <th scope="row">{index}</th>
+                      <th scope="row">{index}/{spellIdToCasts.form}</th>
```

Comment:
I would split these into 2 different columns for a more organized appearance.

Cast # | Form | Spells In Cast

Reply:
Good idea, I've made the change.